[2025-04-18T11:13:32.219+0900] {processor.py:186} INFO - Started process (PID=637856) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:13:32.219+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:13:32.221+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:13:32.221+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:13:32.223+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:13:32.223+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:13:32.223+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:13:32.223+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:13:32.224+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:13:32.299+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.085 seconds
[2025-04-18T11:14:35.390+0900] {processor.py:186} INFO - Started process (PID=638243) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:14:35.392+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:14:35.393+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:14:35.393+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:14:35.396+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:14:35.395+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:14:35.396+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:14:35.396+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:14:35.396+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:14:35.403+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-04-18T11:15:34.486+0900] {processor.py:186} INFO - Started process (PID=638769) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:15:34.488+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:15:34.490+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:15:34.490+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:15:34.494+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:15:34.494+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:15:34.495+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:15:34.494+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:15:34.495+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:15:34.504+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2025-04-18T11:16:34.543+0900] {processor.py:186} INFO - Started process (PID=639368) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:16:34.544+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:16:34.546+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:16:34.546+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:16:34.549+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:16:34.548+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:16:34.549+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:16:34.549+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:16:34.549+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:16:34.556+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-04-18T11:17:33.777+0900] {processor.py:186} INFO - Started process (PID=640049) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:17:33.780+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:17:33.782+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:17:33.782+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:17:33.785+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:17:33.784+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:17:33.785+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:17:33.785+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:17:33.785+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:17:33.793+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2025-04-18T11:18:35.489+0900] {processor.py:186} INFO - Started process (PID=640724) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:18:35.490+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:18:35.492+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:18:35.492+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:18:35.495+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:18:35.494+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:18:35.495+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:18:35.495+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:18:35.495+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:18:35.503+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2025-04-18T11:19:34.521+0900] {processor.py:186} INFO - Started process (PID=641011) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:19:34.524+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:19:34.526+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:19:34.525+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:19:34.529+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:19:34.529+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:19:34.529+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:19:34.529+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:19:34.530+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:19:34.540+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2025-04-18T11:20:35.168+0900] {processor.py:186} INFO - Started process (PID=641304) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:20:35.170+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:20:35.171+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:20:35.171+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:20:35.174+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:20:35.173+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:20:35.174+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:20:35.174+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:20:35.174+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:20:35.182+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-04-18T11:21:36.273+0900] {processor.py:186} INFO - Started process (PID=641595) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:21:36.276+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:21:36.280+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:21:36.279+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:21:36.285+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:21:36.284+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:21:36.285+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:21:36.285+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:21:36.286+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:21:36.300+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2025-04-18T11:22:38.156+0900] {processor.py:186} INFO - Started process (PID=642056) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:22:38.157+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:22:38.159+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:22:38.159+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:22:38.162+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:22:38.162+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:22:38.163+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:22:38.163+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:22:38.163+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:22:38.172+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-04-18T11:23:34.922+0900] {processor.py:186} INFO - Started process (PID=642340) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:23:34.923+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:23:34.925+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:23:34.924+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:23:34.927+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:23:34.927+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:23:34.928+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:23:34.928+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:23:34.928+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:23:34.935+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2025-04-18T11:24:33.396+0900] {processor.py:186} INFO - Started process (PID=642703) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:24:33.398+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:24:33.400+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:24:33.400+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:24:33.403+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:24:33.402+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:24:33.403+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:24:33.403+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:24:33.403+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:24:33.410+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-04-18T11:25:35.922+0900] {processor.py:186} INFO - Started process (PID=643244) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:25:35.925+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T11:25:35.927+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:25:35.926+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:25:35.930+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:25:35.930+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T11:25:35.930+0900] {logging_mixin.py:190} INFO - [2025-04-18T11:25:35.930+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T11:25:35.930+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T11:25:35.938+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2025-04-18T12:03:13.060+0900] {processor.py:186} INFO - Started process (PID=651821) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:03:13.062+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T12:03:13.063+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:03:13.063+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:03:13.146+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:03:13.146+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T12:03:13.146+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:03:13.146+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T12:03:13.147+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:03:13.160+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.105 seconds
[2025-04-18T12:05:19.208+0900] {processor.py:186} INFO - Started process (PID=653649) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:05:19.209+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T12:05:19.210+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:05:19.210+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:05:19.213+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:05:19.212+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T12:05:19.213+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:05:19.213+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T12:05:19.213+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:05:19.228+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2025-04-18T12:05:50.006+0900] {processor.py:186} INFO - Started process (PID=654127) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:05:50.008+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T12:05:50.010+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:05:50.010+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:05:50.014+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:05:50.013+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T12:05:50.014+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:05:50.014+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T12:05:50.014+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:05:50.035+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2025-04-18T12:06:20.639+0900] {processor.py:186} INFO - Started process (PID=654504) to work on /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:06:20.640+0900] {processor.py:914} INFO - Processing file /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2025-04-18T12:06:20.642+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:06:20.642+0900] {dagbag.py:588} INFO - Filling up the DagBag from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:06:20.645+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:06:20.644+0900] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2025-04-18T12:06:20.645+0900] {logging_mixin.py:190} INFO - [2025-04-18T12:06:20.645+0900] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2025-04-18T12:06:20.645+0900] {processor.py:927} WARNING - No viable dags retrieved from /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2025-04-18T12:06:20.664+0900] {processor.py:208} INFO - Processing /home/kim/airflow_tutorial/py_env/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
